xcube/core/store/fs/accessor.py

```diff
@@ -132,9 +132,9 @@ def load_fs(cls, params: Dict[str, Any]) \
                 root,
                 params
             )
-        except (ValueError, ImportError):
-            raise DataStoreError(f"Cannot instantiate"
-                                 f" filesystem {protocol!r}")
+        except (ValueError, ImportError) as error:
+            raise DataStoreError(
+                f"Cannot instantiate filesystem {protocol!r}: {error}")
```

Comment:
It would be nice, if we say that the reason for this is most likely a missing package that provides the `fsspec` implementation.

```suggestion
                f"Cannot instantiate filesystem {protocol!r}"
                " (most likely your environment is missing the"
                " required `fsspec` implementation): {error}"
            )
```

Reply:
Fortunately, the error message that we interpolate from `fsspec` already supplies an informative and constructive error message if there’s a missing package. For example, in the case of `abfs`, the user will see:

> xcube.core.store.error.DataStoreError: Cannot instantiate filesystem 'abfs': Install adlfs to access Azure Datalake Gen2 and Azure Blob Storage

See [here](https://github.com/fsspec/filesystem_spec/blob/ecc576506567b8413a4c5402cbb06b28f1ba27bb/fsspec/registry.py#L62) for the error messages for other protocols -- they’re all similarly helpful. With this in mind, I think that the additional explanation on our side is superfluous if the problem is a missing package, and potentially misleading if it isn’t.